forms.html

```diff
@@ -0,0 +1,102 @@
+        <input type="radio" id="female" value="female" />
+        <label for="other">Other</label>
+        <input type="radio" id="other" value="other" /><br />
+        <label for="date">Date:</label>
+        <input type="date" id="date" name="date" />
+      </fieldset>
+      <fieldset>
+        <legend>Address Information:</legend>
+        <span>Street Address:</span>
+        <textarea name="street-address" rows="2" cols="20"></textarea><br />
+        <span>Country:</span>
+        <input list="countries" name="countries" />
+        <datalist id="countries">
+          <option value="USA"></option>
+          <option value="UK"></option>
+          <option value="Belgium"></option>
+          <option value="France"></option>
+          <option value="Spain"></option>
+        </datalist>
+        <br />
+        <label for="city">City:</label>
+        <input type="text" id="city" name="city" /><br />
+        <label for="postal-code">Postal Code:</label>
+        <input type="number" id="postal-code" name="postal-code" /><br />
+      </fieldset>
+      <fieldset>
+        <legend>Preferences:</legend>
+        <span>Preferred Contact Method:</span>
+        <input type="checkbox" id="email1" name="email1" value="email1" />
+        <label for="email1">Email</label>
+        <input type="checkbox" id="phone" name="phone" value="phone" />
+        <label for="phone">Phone</label>
+        <input
+          type="checkbox"
+          id="post-mail"
+          name="post-mail"
+          value="post-mail"
+        />
+        <label for="post-mail">Postal Mail</label><br />
+        <label for="favcolor">Favorite color:</label>
+        <input type="color" id="favcolor" name="favcolor" value="#ff0000" />
+      </fieldset>
+      <fieldset>
+        <legend>Additional Details:</legend>
+        <span>How did you hear about us?</span>
+        <input list="sources" name="sources" />
+        <datalist id="sources">
+          <option value="Internet"></option>
+          <option value="Linkedin"></option>
+          <option value="Friends"></option>
+          <option value="Advertisement"></option>
+        </datalist>
+        <br />
+        <span>Feedback/Comments:</span>
+        <textarea name="feedback" rows="3" cols="25"></textarea>
+      </fieldset>
+      <fieldset>
+        <legend>Profile:</legend>
+        <span>Upload a Profile Picture:</span>
+        <input type="file" name="picture" id="picture" accept=".jpg, .png" />
+        <label for="picture"></label><br />
+        <span>Website URL:</span>
+        <input type="url" name="web-url" id="web-url" />
+        <label for="web-url"></label><br />
+        <span>Experience Level with HTML:</span>
+        <label for="exp-level"></label>
+        <input
+          type="range"
+          name="exp-level"
+          id="exp-level"
+          min="1"
+          max="10"
+          oninput="displayValue.value=experience.value"
+        />
```

Comment:
Try to use output here

Reply:
Point taken, changes implemented. Thank you!